templates/home/downloads.html.twig

```diff
@@ -0,0 +1,20 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="247" height="20">
```

Comment:
this file is useless if you use the approach described here: https://github.com/PUGX/badge-poser/pull/221#issuecomment-383836145